authors/apps/authentication/utils.py

```diff
@@ -6,28 +6,30 @@
 from django.template.loader import render_to_string
 
 # Method to Send emails
-def send_link(email):
+
+
+def send_link(email, subject, template, url):
     payload = {'email':  email,
-                "iat": datetime.now(),
-                "exp": datetime.utcnow()
-                + timedelta(minutes=5)}
+               "iat": datetime.now(),
+               "exp": datetime.utcnow()
+               + timedelta(minutes=5)}
     token = jwt.encode(payload,
-                        settings.SECRET_KEY,
-                        algorithm='HS256').decode('utf-8')
+                       settings.SECRET_KEY,
+                       algorithm='HS256').decode('utf-8')
```

Comment:
You don't have to generate token in this function again as it will be passed in by the view if needed. Check my other comments to understand better.